hadoop-ozone/dist/src/main/compose/ozone/prometheus.yml

```
@@ -33,15 +32,19 @@ scrape_configs:
        labels:
          component: s3g
      - targets:
-        - "ozone_datanode_1:9882"
-        - "ozone_datanode_2:9882"
-        - "ozone_datanode_3:9882"
-        - "ozone_datanode_4:9882"
-        - "ozone_datanode_5:9882"
-        - "ozone_datanode_6:9882"
-        - "ozone_datanode_7:9882"
-        - "ozone_datanode_8:9882"
-        - "ozone_datanode_9:9882"
-        - "ozone_datanode_10:9882"
+        - "ozone-datanode-1:9882"
+        - "ozone-datanode-2:9882"
+        - "ozone-datanode-3:9882"
+        - "ozone-datanode-4:9882"
+        - "ozone-datanode-5:9882"
+        - "ozone-datanode-6:9882"
+        - "ozone-datanode-7:9882"
+        - "ozone-datanode-8:9882"
+        - "ozone-datanode-9:9882"
+        - "ozone-datanode-10:9882"
```

Both formats are correct, just for different versions of Docker Compose (v1 / v2).  So by changing it to v2 format, the patch introduces the same problem for v1.

To make it work for both versions, please add the new targets instead of replacing the existing ones.

```suggestion
        - "ozone_datanode_1:9882"
        - "ozone_datanode_2:9882"
        - "ozone_datanode_3:9882"
        - "ozone_datanode_4:9882"
        - "ozone_datanode_5:9882"
        - "ozone_datanode_6:9882"
        - "ozone_datanode_7:9882"
        - "ozone_datanode_8:9882"
        - "ozone_datanode_9:9882"
        - "ozone_datanode_10:9882"
        - "ozone-datanode-1:9882"
        - "ozone-datanode-2:9882"
        - "ozone-datanode-3:9882"
        - "ozone-datanode-4:9882"
        - "ozone-datanode-5:9882"
        - "ozone-datanode-6:9882"
        - "ozone-datanode-7:9882"
        - "ozone-datanode-8:9882"
        - "ozone-datanode-9:9882"
        - "ozone-datanode-10:9882"
```